docs/src/calibration_features.md

```diff
@@ -1,28 +1,59 @@
+```
+
+Inside the calibration example, there are examples of the `config.jl` file, the `run_calibration.jl` file, and `run_model.jl`.
+
+In `config.jl`, we define the function `get_config()`, which creates the configuration dictionary. The keys of this dictionary are associated with all the elements that are needed for the calibration process, and they are retrieved using the following functions:
+- `get_prior_config()`: Sets up the priors' configuration.
+- `get_observations_config()`: Specifies the reference data.  
+- `get_process_config()`: Defines the method for the calibration and the associated parameters.
+- `get_model_config()`: Defines the model setup.
+- `get_stats_config()`: Specifies the statistical methods used on the reference data and simulation results.
+
+After creating the configuration file, running the calibrations using the `run_calibration.jl` file becomes very straightforward. 
+By importing the `/KinematicDriver/src/CalibrateCMP` folder and including the configuration file, you can use the appropriate functions to construct the priors and get the reference observations. Depending on the calibration method that you choose, the `calibrate()` and `get_results()` functions allow you to perform the calibration steps and obtain the final set of optimal parameters. 
+When choosing the EKI or UKI methods, `calibrate()` outputs the new updated ensemble and the optimal parameters for the calibration problem, that are given by the means of the last ensembles. To obtain the optimal values in the constrained space as well as the covariance connected to the distribution that describes the solution of the inversion problem, you can simply call the `get_results()` function.
 
-- **Prior Settings**: Defines the prior distributions of parameters.
-- **Observations**: Specifies the reference data against which simulations are compared.
-- **Calibration Process**: Outlines the methods and approaches used for calibration.
-- **Model Parameters**: Details parameters that affect the model's behavior.
-- **Statistics**: Specifies the statistical methods to be applied on the data and simulation results.
+in the following animation, you can visualize the output of `run_calibration.jl`, which illustrates the ensemble convergence process as the number of iterations increases:
 
-## Example and Documentation
+![Ensemble convergence](assets/ensemble_convergence.gif)
 
-An example of the configuration file for a simple microphysics calibration can be found at
-`test/experiments/calibrations/config.jl`,
-and the program to run the calibration at 
-`test/experiments/calibrations/run_calibration.jl`.
\ No newline at end of file
+Finally, by running the `run_model.jl`, you can also visualize the final contour plots of the cloud water mixing ratio and rain water mixing ratio  that are obtained with KiD after the calibration process, and compare them with the particle-based method results.
```

Comment:
..., you can see contour plots of the cloud water mixing ratio and rain water mixing ratio that are obtained with KiD, and compare them with reference data.